### PR TITLE
Meter.Get with registration

### PR DESCRIPTION
**Pull request by LopatkinEvgeniy**
* Add registration in Get method
* Remove Counter interface

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ func ExampleWriteToFile() {
 	metrics := gometer.New()
 	metrics.SetFormatter(gometer.NewFormatter("\n"))
 
+	// write metrics to file periodically.
```
Comment (dshil) [inline README.md]: What's the purpose of this comment? We have a description section for it.

Comment (LopatkinEvgeniy) [reply README.md]: I just copied it from ./_examples/write_to_file_test.go. So I'd better remove it from here and from there.

Comment (dshil) [reply README.md]: Oops, sorry, I thought that we don't have it.

**Comment by dshil** [comment]
LGTM

**Pull request merged**